README.md

```diff
@@ -116,6 +111,19 @@ error: unsupported operation: can't call foreign function: bind
             performed an operation that the interpreter does not support
 ```
 
+### Cross-interpretation: running for different targets
+
+Miri cannot just run a binary or test suite for your host target, it can also
```

Comment:
```suggestion
Miri can not only run a binary or test suite for your host target, it can also
```

Reply:
AFAIK "cannot" is usually written together, why do you suggest to split it?

Reply:
Saying 'cannot just' sounds like 'this requires extra steps' (e.g. 'You cannot just use 'cargo build' to compile miri, you must use './miri build').

Reply:
Oh I see... thanks.